homework-14/main.js

```diff
@@ -0,0 +1,121 @@
+            }
+        }
+    })
+
+    const userValues = document.querySelector('.value-input');
+    const resultValue = document.querySelector('.result-value');
+
+    function vowelOne(value) {
+        const vowels = 'aeiouy';
+        let result = '';
+
+        for (let symbol of value) {
+            if (vowels.includes(symbol)) {
+                result += '1';
+            } else {
+                result += '0';
+            }
+        }
+        return result;
+    }
+    document.querySelector('.value-submit').addEventListener('click', (event) => {
+        event.preventDefault();
+        if (userValues.value === '') {
+            resultValue.innerHTML = `Result is't valid.`;
+        } else {
+            resultValue.innerHTML = `Result: ${vowelOne(userValues.value)}.`;
+        }
+    })
+
+    const userValues3 = document.querySelector('.value3-input');
+    const resultReversedValue3 = document.querySelector('.result-value3');
+
+    function reversedValue(value) {
+        let symbol = value.split(' ');
+        let reversedValue = '';
+        let result = '';
+
+        for (let i = 0; i < symbol.length; i++) {
+            if (symbol[i].length >= 2) {
+                reversedValue = symbol[i].split('').reverse();
+                result += `${reversedValue.join('')} `;
+            } else {
+                result += `${symbol[i]}`;
+            }
+        }
+
+        return result;
+    }
+
+    document.querySelector('.value3-submit').addEventListener('click', (event) => {
+        event.preventDefault();
+        if (userValues3.value === '') {
+            resultReversedValue3.innerHTML = `Result is't valid.`;
+        } else {
+            resultReversedValue3.innerHTML = `Result: ${reversedValue(userValues3.value)}`;
+        }
+    })
+    const userNumbers = document.querySelector('.value4-input');
+    const resultNumbers = document.querySelector('.result-value4');
+    let numbersArray = [];
+
+    function getHighestAndLowestNumbers(value) {
+        numbersArray = value.split("");
+        let maxNumber = (a, b) => {
+                return a > b ? a : b;
+            },
+            minNumber = (a, b) => {
+                return a < b ? a : b;
+            };
+        resultNumbers.innerHTML = `Result: max ${numbersArray.reduce(maxNumber)}, min ${numbersArray.reduce(minNumber)}.`;
+
+    }
+
+    document.querySelector('.value4-submit').addEventListener('click', (event) => {
+        event.preventDefault();
+        if (userNumbers.value === '') {
+            resultNumbers.innerHTML = `Result is't valid.`;
+        } else {
+            getHighestAndLowestNumbers(userNumbers.value);
+        }
+    })
+
+    const userNumbersArray = document.querySelector('.value5-input');
+    const resultNumbersPhone = document.querySelector('.result-value5');
+    let numbers = [];
+
+    function createPhoneNumber(value) {
```

Comment:
It works with strings
<img width="789" alt="image" src="https://user-images.githubusercontent.com/84010501/203496685-d67ff2bd-7b6b-4910-853a-6eb678185682.png">

And with empty values
<img width="789" alt="image" src="https://user-images.githubusercontent.com/84010501/203496772-054d9be8-78fa-4f89-9e09-d9c280f3c7b9.png">